commands/issue.go

```diff
@@ -114,8 +114,10 @@ With no arguments, show a list of open issues.
 		of "always" (default for '--color'), "never", or "auto" (default).
 
 	-m, --message <MESSAGE>
-		The text up to the first blank line in <MESSAGE> is treated as the issue
-		title, and the rest is used as issue description in Markdown format.
+		An issue title and an issue description will be read from <MESSAGE>.
+		These should be separated by a blank line (ie: a line with no contents).
```

Comment:
No, sorry, I don't believe this improves upon the old text, which reads:

> The text up to the first blank line in \<MESSAGE> is treated as the issue title, and the rest is used as issue description in Markdown format.

It looks to me that you have just expressed the same concept in a slightly different way. I don't see this as an improvement to our documentation.